packages/flutter_chat/lib/src/screens/chat_detail/chat_detail_screen.dart

```diff
@@ -136,14 +151,14 @@ class _AppBar extends StatelessWidget implements PreferredSizeWidget {
       iconTheme: theme.appBarTheme.iconTheme ??
           const IconThemeData(color: Colors.white),
       centerTitle: true,
-      leading: GestureDetector(
-        onTap: () {
-          Navigator.popUntil(context, (route) => route.isFirst);
-        },
-        child: const Icon(
-          Icons.arrow_back_ios,
-        ),
-      ),
+      leading: onPressBack == null
+          ? null
+          : GestureDetector(
```

Comment:
If you touch a gesturedetector, at least change it to a proper semantic enabled widget, like an inkwell